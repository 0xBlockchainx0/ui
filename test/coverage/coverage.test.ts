import initStoryshots, {
  Stories2SnapsConverter
} from '@storybook/addon-storyshots'
const coverageTest = {
  suite: 'Decentraland UI - Coverage Tests',
  stories2snapsConverter: new Stories2SnapsConverter({
    snapshotsDirName: '__coverage_snapshots__',
    snapshotExtension: '.storyshot',
    storiesExtensions: ['.js', '.jsx', '.ts', '.tsx']
  })
}
initStoryshots(coverageTest)
