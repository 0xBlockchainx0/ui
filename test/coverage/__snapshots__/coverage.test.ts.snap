// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Decentraland UI - Coverage Tests Address Full length 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <span
      className="dcl address"
    >
      0x68FFc53C43C65C8Dd778969320e21B85b10363cE
    </span>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Address In a paragraph 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <p>
      Your address is 
      <strong
        className="dcl address"
      >
        0x68...63cE
      </strong>
      .
    </p>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Address Plain 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <span
      className="dcl address"
    >
      0x68...63cE
    </span>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Address Strong 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <strong
      className="dcl address"
    >
      0x68...63cE
    </strong>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Address Tooltip 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <span
      className="dcl address"
      data-balloon="0x68FFc53C43C65C8Dd778969320e21B85b10363cE"
      data-balloon-pos="up"
    >
      0x68...63cE
    </span>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Blockie In a paragraph 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <p>
      You have transfered 
      <div
        className="ui tiny header dcl mana"
      >
        <i
          className="symbol"
        >
          ⏣
        </i>
        1,000
      </div>
       to
       
      <span
        className="dcl blockie-wrapper"
      >
        <canvas
          className="dcl blockie tiny"
        />
        <span
          className="dcl blockie-children"
        >
          <strong
            className="dcl address"
          >
            0x68...63cE
          </strong>
        </span>
      </span>
      .
    </p>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Blockie Scales 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <canvas
      className="dcl blockie mini"
    />
    <canvas
      className="dcl blockie tiny"
    />
    <canvas
      className="dcl blockie small"
    />
    <canvas
      className="dcl blockie"
    />
    <canvas
      className="dcl blockie"
    />
    <canvas
      className="dcl blockie"
    />
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Blockie Single 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <canvas
      className="dcl blockie small"
    />
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Button Actions 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <button
      className="ui primary button"
      onClick={[Function]}
      role="button"
      style={
        Object {
          "minWidth": 190,
        }
      }
    >
      Vote Now
    </button>
    <button
      className="ui button"
      onClick={[Function]}
      role="button"
      style={
        Object {
          "minWidth": 190,
        }
      }
    >
      Cancel
    </button>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Button Primary 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <button
      className="ui primary button"
      onClick={[Function]}
      role="button"
    >
      Vote Now
    </button>
    <button
      className="ui primary disabled button"
      disabled={true}
      onClick={[Function]}
      role="button"
      tabIndex={-1}
    >
      Vote Now
    </button>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Button Secondary 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <button
      className="ui button"
      onClick={[Function]}
      role="button"
    >
      Cancel
    </button>
    <button
      className="ui disabled button"
      disabled={true}
      onClick={[Function]}
      role="button"
      tabIndex={-1}
    >
      Cancel
    </button>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Button Sizes 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <button
      className="ui massive primary button"
      onClick={[Function]}
      role="button"
    >
      Massive
    </button>
    <button
      className="ui huge primary button"
      onClick={[Function]}
      role="button"
    >
      Huge
    </button>
    <button
      className="ui large primary button"
      onClick={[Function]}
      role="button"
    >
      Large
    </button>
    <button
      className="ui medium primary button"
      onClick={[Function]}
      role="button"
    >
      Medium
    </button>
    <button
      className="ui small primary button"
      onClick={[Function]}
      role="button"
    >
      Small
    </button>
    <button
      className="ui tiny primary button"
      onClick={[Function]}
      role="button"
    >
      Tiny
    </button>
    <button
      className="ui mini primary button"
      onClick={[Function]}
      role="button"
    >
      Mini
    </button>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Header Subheader + Header 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="ui sub header"
    >
      Votes
    </div>
    <div
      className="ui large header"
    >
      20,212
    </div>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Mana Black 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="ui small header dcl mana black"
    >
      <i
        className="symbol"
      >
        ⏣
      </i>
      20
    </div>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Mana In a paragraph 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <p>
      You have voted with 
      <div
        className="ui small header dcl mana"
      >
        <i
          className="symbol"
        >
          ⏣
        </i>
        1,000
      </div>
      .
    </p>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Mana Sizes 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="ui huge header dcl mana"
    >
      <i
        className="symbol"
      >
        ⏣
      </i>
      1,000
    </div>
    <div
      className="ui large header dcl mana"
    >
      <i
        className="symbol"
      >
        ⏣
      </i>
      1,000
    </div>
    <div
      className="ui medium header dcl mana"
    >
      <i
        className="symbol"
      >
        ⏣
      </i>
      1,000
    </div>
    <div
      className="ui small header dcl mana"
    >
      <i
        className="symbol"
      >
        ⏣
      </i>
      1,000
    </div>
    <div
      className="ui tiny header dcl mana"
    >
      <i
        className="symbol"
      >
        ⏣
      </i>
      1,000
    </div>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Mana Symbol + MANA 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="ui header dcl mana"
    >
      <i
        className="symbol"
      >
        ⏣
      </i>
      MANA
    </div>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Mana Symbol 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="ui header dcl mana"
    >
      <i
        className="symbol"
      >
        ⏣
      </i>
    </div>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Mana Total voted 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="ui sub header"
    >
      Total voted
    </div>
    <div
      className="ui large header dcl mana"
    >
      <i
        className="symbol"
      >
        ⏣
      </i>
      1,235,345
    </div>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Radio Checked 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="ui checked radio checkbox"
      onChange={[Function]}
      onClick={[Function]}
      onMouseDown={[Function]}
    >
      <input
        checked={true}
        className="hidden"
        onClick={[Function]}
        readOnly={true}
        tabIndex={0}
        type="radio"
      />
      <label>
        Yes
      </label>
    </div>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Radio Options 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="ui checked radio checkbox"
      onChange={[Function]}
      onClick={[Function]}
      onMouseDown={[Function]}
    >
      <input
        checked={true}
        className="hidden"
        onClick={[Function]}
        readOnly={true}
        tabIndex={0}
        type="radio"
      />
      <label>
        Yes
      </label>
    </div>
    <div
      className="ui radio checkbox"
      onChange={[Function]}
      onClick={[Function]}
      onMouseDown={[Function]}
    >
      <input
        checked={false}
        className="hidden"
        onClick={[Function]}
        readOnly={true}
        tabIndex={0}
        type="radio"
      />
      <label>
        No
      </label>
    </div>
    <div
      className="ui radio checkbox"
      onChange={[Function]}
      onClick={[Function]}
      onMouseDown={[Function]}
    >
      <input
        checked={false}
        className="hidden"
        onClick={[Function]}
        readOnly={true}
        tabIndex={0}
        type="radio"
      />
      <label>
        Abstain
      </label>
    </div>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Radio Unchecked 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      className="ui radio checkbox"
      onChange={[Function]}
      onClick={[Function]}
      onMouseDown={[Function]}
    >
      <input
        checked={false}
        className="hidden"
        onClick={[Function]}
        readOnly={true}
        tabIndex={0}
        type="radio"
      />
      <label>
        No
      </label>
    </div>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Stats Poll Results 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <span
      className="dcl stats"
    >
      <div
        className="ui sub header"
      >
        Token
      </div>
      <div
        className="ui large header dcl mana"
      >
        <i
          className="symbol"
        >
          ⏣
        </i>
        MANA
      </div>
    </span>
    <span
      className="dcl stats"
    >
      <div
        className="ui sub header"
      >
        Total Voted
      </div>
      <div
        className="ui large header dcl mana"
      >
        <i
          className="symbol"
        >
          ⏣
        </i>
        1,235,345
      </div>
    </span>
    <span
      className="dcl stats"
    >
      <div
        className="ui sub header"
      >
        Total Votes
      </div>
      <div
        className="ui large header"
      >
        20
      </div>
    </span>
    <span
      className="dcl stats"
    >
      <div
        className="ui sub header"
      >
        Parcial Result
      </div>
      <div
        className="ui large header"
      >
        Yes
      </div>
    </span>
    <span
      className="dcl stats"
    >
      <div
        className="ui sub header"
      >
        Time Left
      </div>
      <div
        className="ui large header"
      >
        1 week
      </div>
    </span>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Stats Single Result 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <span
      className="dcl stats"
    >
      <div
        className="ui sub header"
      >
        Total Voted
      </div>
      <div
        className="ui large header dcl mana"
      >
        <i
          className="symbol"
        >
          ⏣
        </i>
        1,235,345
      </div>
    </span>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Table Votes 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      style={
        Object {
          "width": 1024,
        }
      }
    >
      <div
        className="ui header"
      >
        Votes
      </div>
      <table
        className="ui basic table"
      >
        <thead
          className=""
        >
          <tr
            className=""
          >
            <th
              className=""
            >
              When
            </th>
            <th
              className=""
            >
              Address
            </th>
            <th
              className=""
            >
              Amount
            </th>
            <th
              className=""
            >
              Vote
            </th>
          </tr>
        </thead>
        <tbody
          className=""
        >
          <tr
            className=""
          >
            <td
              className=""
            >
              Feb 10, 2018
            </td>
            <td
              className=""
            >
              <span
                className="dcl blockie-wrapper"
              >
                <canvas
                  className="dcl blockie tiny"
                />
                <span
                  className="dcl blockie-children"
                >
                   
                  <span
                    className="dcl address"
                  >
                    0x0a...36ea
                  </span>
                </span>
              </span>
            </td>
            <td
              className=""
            >
              <div
                className="ui small header dcl mana black"
              >
                <i
                  className="symbol"
                >
                  ⏣
                </i>
              </div>
              180
            </td>
            <td
              className=""
            >
              Yes
            </td>
          </tr>
          <tr
            className=""
          >
            <td
              className=""
            >
              Feb 11, 2018
            </td>
            <td
              className=""
            >
              <span
                className="dcl blockie-wrapper"
              >
                <canvas
                  className="dcl blockie tiny"
                />
                <span
                  className="dcl blockie-children"
                >
                   
                  <span
                    className="dcl address"
                  >
                    0x39...79bc
                  </span>
                </span>
              </span>
            </td>
            <td
              className=""
            >
              <div
                className="ui small header dcl mana black"
              >
                <i
                  className="symbol"
                >
                  ⏣
                </i>
              </div>
              180
            </td>
            <td
              className=""
            >
              Yes
            </td>
          </tr>
          <tr
            className=""
          >
            <td
              className=""
            >
              Feb 12, 2018
            </td>
            <td
              className=""
            >
              <span
                className="dcl blockie-wrapper"
              >
                <canvas
                  className="dcl blockie tiny"
                />
                <span
                  className="dcl blockie-children"
                >
                   
                  <span
                    className="dcl address"
                  >
                    0xba...cc3f
                  </span>
                </span>
              </span>
            </td>
            <td
              className=""
            >
              <div
                className="ui small header dcl mana black"
              >
                <i
                  className="symbol"
                >
                  ⏣
                </i>
              </div>
              120
            </td>
            <td
              className=""
            >
              Yes
            </td>
          </tr>
          <tr
            className=""
          >
            <td
              className=""
            >
              Feb 13, 2018
            </td>
            <td
              className=""
            >
              <span
                className="dcl blockie-wrapper"
              >
                <canvas
                  className="dcl blockie tiny"
                />
                <span
                  className="dcl blockie-children"
                >
                   
                  <span
                    className="dcl address"
                  >
                    0x82...19a5
                  </span>
                </span>
              </span>
            </td>
            <td
              className=""
            >
              <div
                className="ui small header dcl mana black"
              >
                <i
                  className="symbol"
                >
                  ⏣
                </i>
              </div>
              80
            </td>
            <td
              className=""
            >
              No
            </td>
          </tr>
          <tr
            className=""
          >
            <td
              className=""
            >
              Feb 14, 2018
            </td>
            <td
              className=""
            >
              <span
                className="dcl blockie-wrapper"
              >
                <canvas
                  className="dcl blockie tiny"
                />
                <span
                  className="dcl blockie-children"
                >
                   
                  <span
                    className="dcl address"
                  >
                    0xea...0466
                  </span>
                </span>
              </span>
            </td>
            <td
              className=""
            >
              <div
                className="ui small header dcl mana black"
              >
                <i
                  className="symbol"
                >
                  ⏣
                </i>
              </div>
              120
            </td>
            <td
              className=""
            >
              Yes
            </td>
          </tr>
          <tr
            className=""
          >
            <td
              className=""
            >
              Feb 15, 2018
            </td>
            <td
              className=""
            >
              <span
                className="dcl blockie-wrapper"
              >
                <canvas
                  className="dcl blockie tiny"
                />
                <span
                  className="dcl blockie-children"
                >
                   
                  <span
                    className="dcl address"
                  >
                    0xd2...d28b
                  </span>
                </span>
              </span>
            </td>
            <td
              className=""
            >
              <div
                className="ui small header dcl mana black"
              >
                <i
                  className="symbol"
                >
                  ⏣
                </i>
              </div>
              50
            </td>
            <td
              className=""
            >
              Yes
            </td>
          </tr>
          <tr
            className=""
          >
            <td
              className=""
            >
              Feb 16, 2018
            </td>
            <td
              className=""
            >
              <span
                className="dcl blockie-wrapper"
              >
                <canvas
                  className="dcl blockie tiny"
                />
                <span
                  className="dcl blockie-children"
                >
                   
                  <span
                    className="dcl address"
                  >
                    0x72...51f6
                  </span>
                </span>
              </span>
            </td>
            <td
              className=""
            >
              <div
                className="ui small header dcl mana black"
              >
                <i
                  className="symbol"
                >
                  ⏣
                </i>
              </div>
              50
            </td>
            <td
              className=""
            >
              Yes
            </td>
          </tr>
          <tr
            className=""
          >
            <td
              className=""
            >
              Feb 17, 2018
            </td>
            <td
              className=""
            >
              <span
                className="dcl blockie-wrapper"
              >
                <canvas
                  className="dcl blockie tiny"
                />
                <span
                  className="dcl blockie-children"
                >
                   
                  <span
                    className="dcl address"
                  >
                    0x7b...2040
                  </span>
                </span>
              </span>
            </td>
            <td
              className=""
            >
              <div
                className="ui small header dcl mana black"
              >
                <i
                  className="symbol"
                >
                  ⏣
                </i>
              </div>
              30
            </td>
            <td
              className=""
            >
              Yes
            </td>
          </tr>
          <tr
            className=""
          >
            <td
              className=""
            >
              Feb 18, 2018
            </td>
            <td
              className=""
            >
              <span
                className="dcl blockie-wrapper"
              >
                <canvas
                  className="dcl blockie tiny"
                />
                <span
                  className="dcl blockie-children"
                >
                   
                  <span
                    className="dcl address"
                  >
                    0xff...e688
                  </span>
                </span>
              </span>
            </td>
            <td
              className=""
            >
              <div
                className="ui small header dcl mana black"
              >
                <i
                  className="symbol"
                >
                  ⏣
                </i>
              </div>
              160
            </td>
            <td
              className=""
            >
              No
            </td>
          </tr>
          <tr
            className=""
          >
            <td
              className=""
            >
              Feb 19, 2018
            </td>
            <td
              className=""
            >
              <span
                className="dcl blockie-wrapper"
              >
                <canvas
                  className="dcl blockie tiny"
                />
                <span
                  className="dcl blockie-children"
                >
                   
                  <span
                    className="dcl address"
                  >
                    0xce...8b94
                  </span>
                </span>
              </span>
            </td>
            <td
              className=""
            >
              <div
                className="ui small header dcl mana black"
              >
                <i
                  className="symbol"
                >
                  ⏣
                </i>
              </div>
              40
            </td>
            <td
              className=""
            >
              No
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;

exports[`Decentraland UI - Coverage Tests Tooltip Hover to show 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
        "maxHeight": "100%",
      }
    }
  >
    <div
      style={
        Object {
          "maxWidth": 100,
        }
      }
    >
      <div
        className="ui large header dcl mana"
        data-balloon="MANA"
        data-balloon-pos="up"
      >
        <i
          className="symbol"
        >
          ⏣
        </i>
      </div>
    </div>
  </div>
</div>
`;
